packages/components/src/mobile/color-settings/index.native.js

```diff
@@ -1,240 +1,82 @@
-					}
-				/>
-			);
-		}
-		return (
-			<View style={ styles.footer }>
-				<View style={ styles.flex }>
-					{ currentValue && (
-						<ColorIndicator
-							color={ currentValue }
-							style={ styles.colorIndicator }
-						/>
-					) }
-				</View>
-				<Text
-					style={ styles.selectColorText }
-					maxFontSizeMultiplier={ 2 }
-				>
-					{ __( 'Select a color' ) }
-				</Text>
-				<View style={ styles.flex } />
-			</View>
-		);
-	}
+	const GradientPickerView = useRef( () => (
+		<BottomSheet.NavigationScreen>
+			<GradientPickerScreen />
+		</BottomSheet.NavigationScreen>
+	) );
 
 	return (
-		<View renderToHardwareTextureAndroid>
-			{ isCustomScreen && (
-				<View>
-					<ColorPicker
-						shouldEnableBottomSheetScroll={
-							shouldEnableBottomSheetScroll
-						}
-						shouldDisableBottomSheetMaxHeight={
-							shouldDisableBottomSheetMaxHeight
-						}
-						setColor={ setColor }
-						activeColor={ currentValue }
-						isGradientColor={ isGradientColor }
-						onNavigationBack={ () => onCustomScreenToggle( false ) }
-						onCloseBottomSheet={ onCloseBottomSheet }
-						isBottomSheetContentScrolling={
-							isBottomSheetContentScrolling
-						}
-					/>
-				</View>
-			) }
-			{ ! isCustomScreen && ! isCustomGradientScreen && (
-				<View>
-					<NavigationHeader
-						screen={ label }
-						leftButtonOnPress={ () =>
-							onReplaceSubsheet( subsheets[ 0 ] )
-						}
-					/>
-					<ColorPalette
-						setColor={ setColor }
-						activeColor={ currentValue }
-						isGradientColor={ isGradientColor }
-						currentSegment={ currentSegment }
-						isCustomScreen={ isCustomScreen }
-						onCustomPress={ onCustomPress }
-						shouldEnableBottomSheetScroll={
-							shouldEnableBottomSheetScroll
-						}
-						defaultSettings={ defaultSettings }
-					/>
-					{ isCustomGadientShown && (
-						<>
-							<View style={ horizontalSeparatorStyle } />
-							<PanelBody>
-								<ColorControl
-									label={ __( 'Customize Gradient' ) }
-									onPress={ () =>
-										onCustomGradientScreenToggle( true )
-									}
-									withColorIndicator={ false }
-								/>
-							</PanelBody>
-						</>
-					) }
-					<View style={ horizontalSeparatorStyle } />
-					{ getFooter() }
-				</View>
-			) }
-			{ isCustomGradientScreen && (
-				<View>
-					<NavigationHeader
-						screen={ __( 'Customize Gradient' ) }
-						leftButtonOnPress={ () =>
-							onCustomGradientScreenToggle( false )
-						}
-					/>
-					<CustomGradientPicker
-						setColor={ setColor }
-						currentValue={ currentValue }
-						isGradientColor={ isGradientColor }
-					/>
-				</View>
-			) }
-		</View>
+		<BottomSheet.NavigationContainer>
+			<Stack.Navigator
+				screenOptions={ {
+					headerShown: false,
+					gestureEnabled: false,
+				} }
+			>
+				<Stack.Screen
+					name={ colorsUtils.screens.palette }
+					component={ PaletteScreenView.current }
+					options={ BottomSheet.NavigationScreen.options }
```

Comment:
It seems like we use this in all the `Screen`s. I wonder if we can DRY this up somehow. Not a blocker for this iteration, but perhaps a component that includes these bottomsheet screen options if we are always using the defaults?